### ISSUE TITLE
Button Text Over-Broken

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#31]
# Description
```
Please provide a short description of what you have encountered below.
```
Fix button text over-breaking effect.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```

![screenshot-2022-12-25-11-43-59](https://user-images.githubusercontent.com/7259572/209456313-273db0d4-c434-4951-8db9-0cf92697f890.jpg)

# Current Behavior
```
Please specify the current behavior of software below.
```
![screenshot-2022-12-25-11-27-18](https://user-images.githubusercontent.com/7259572/209456294-145006e9-7a9e-407b-afd0-cd3b7ac7a429.jpg)

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Caused by `overflow-wrap` and `word-break` not being `normal`.

**Comment by hollowaykeanho** [comment]
fixed in 7377a968e1fc54c5b9d56c8fcc81c4ce341b66bb

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
